modules/update-common-terms.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: REFERENCE
+[id="update-common-terms_{context}"]
+= Common terms
+
+Control plane:: The _control plane_, which is composed of control plane machines, manages the {product-title} cluster. The control plane machines manage workloads on the compute machines, which are also known as worker machines.
+
+Cluster Version Operator:: The _Cluster Version Operator_ (CVO) checks with the OSUS to view valid updates and update paths based on your current component versions and information in the graph.
+
+Machine Config Operator:: The _Machine Config Operator_ (MCO) is a cluster-level Operator that manages the operating system and machine configurations. Through the MCO, platform administrators can configure and update systemd, CRI-O and Kubelet, the kernel, NetworkManager, and other system features on the nodes.
++
+For more information, see link:https://docs.openshift.com/container-platform/latest/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks[Machine config overview].
+
+Recommended update edge:: A _recommended update edge_ is a recommended update between {product-title} releases.  Whether a given update is recommended can depend on the cluster's configured channel, current version, known bugs, and other information.
+
+Extended Update Support:: All post-4.7 even-numbered minor releases are labeled as _Extended Update Support_ (EUS) releases. These releases introduce a verified update path between EUS releases, permitting customers to streamline updates of worker nodes and formulate update strategies of EUS-to-EUS {product-title} releases that will cause fewer reboots of nodes.
++
+For more information, see link:https://access.redhat.com/support/policy/updates/openshift-eus[Red Hat OpenShift Extended Update Support (EUS) Overview].
+
+OpenShift Update Service:: The _OpenShift Update Service_ (OSUS) provides over-the-air updates to {product-title}, including to {op-system-first}. It provides a graph, or diagram, that contains the vertices of component Operators and the edges that connect them.
++
+For more information, see link:https://docs.openshift.com/container-platform/latest/updating/understanding-the-update-service.html#update-service-overview_understanding-the-update-service[About the OpenShift Update Service].
+
+Channels:: _Channels_ provide the appropriate release versions for your cluster. Update channels enable you to choose an update strategy. The channels are tied to minor versions of {product-title}.
```

Comment:
nit: might be worth flipping the order here to start out with what you give and then move to what you get.  Maybe:

> Channels:: _Channels_ declare an update strategy tied to minor versions of {product-title}.  The OSUS uses this configured strategy to recommend update edges consistent with that strategy.